pkg/backup/snapshotter/snapshotter_aws.go

```diff
@@ -105,8 +104,10 @@ func (s *AWSSnapshotter) AddVolumeTags(pvs []*corev1.PersistentVolume) error {
 	for _, pv := range pvs {
 		tags := make(map[string]string)
 		tags[PVCTagKey] = pv.GetName()
-		tags[PodTagKey] = pv.GetAnnotations()[label.AnnPodNameKey]
-		tags[PodNSTagKey] = pv.GetLabels()[label.NamespaceLabelKey]
+		if pv.Spec.ClaimRef != nil {
+			tags[PvcNameTagKey] = pv.Spec.ClaimRef.Name
+			tags[PvcNSTagKey] = pv.Spec.ClaimRef.Namespace
```

Comment:
The code seems correct But the screenshot in the PR does NOT show the tag "kubernetes.io/created-for/pvc/namespace", but rather shows "snapshot/kubernetes.io/created-for/pvc/namespace".